server/room-battle.ts

```diff
@@ -1111,6 +1114,16 @@ export class RoomBattle extends RoomGames.RoomGame {
 		// @ts-ignore
 		this.room = null;
 	}
+	onChatMessage(message: string, user: User) {
+		void this.stream.write(`>chat ${user.getIdentity(this.room.roomid)}|${message}`);
+		return ' ';
+	}
```

Comment:
Instead of making the entire chat message roundtrip from battles, I would probably just ignore chat messages coming from the subprocess.

Reply:
Might as well make chat messages include timestamps, too.